From 0e140563e56497ee555af9479c6f0a3557aac15f Mon Sep 17 00:00:00 2001
From: Daniel Templeton <templedf@apache.org>
Date: Wed, 30 Nov 2016 11:22:21 -0800
Subject: [PATCH 2090/2848] YARN-5942. "Overridden" is misspelled as
 "overriden" in FairScheduler.md (Contributed by
 Heather Sutherland via Daniel Templeton)

(cherry picked from commit 57194889fc39a36dc0697b29eafc4ac07ee42d00)
(cherry picked from commit 3013b028baaaec79270f73c6ac008e0c4036b02d)

Change-Id: I5aad07ac53289fdb61039b7f7a1331fe95f8090e
---
 .../src/site/apt/FairScheduler.apt.vm              |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm
index 6a203b2..635f2a9 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm
@@ -309,16 +309,16 @@ Allocation file format
    element in root queue.
 
  * <<A queueMaxAppsDefault element>>, which sets the default running app limit
-   for queues; overriden by maxRunningApps element in each queue.
+   for queues; overridden by maxRunningApps element in each queue.
 
  * <<A queueMaxResourcesDefault element>>: which sets the default max resource limit
-   for queue; overriden by maxResources element in each queue.
+   for queue; overridden by maxResources element in each queue.
 
  * <<A queueMaxAMShareDefault element>>, which sets the default AM resource
-   limit for queue; overriden by maxAMShare element in each queue.
+   limit for queue; overridden by maxAMShare element in each queue.
 
  * <<A defaultQueueSchedulingPolicy element>>, which sets the default scheduling
-   policy for queues; overriden by the schedulingPolicy element in each queue
+   policy for queues; overridden by the schedulingPolicy element in each queue
    if specified. Defaults to "fair".
 
  * <<A queuePlacementPolicy element>>, which contains a list of rule elements
-- 
1.7.9.5

